submissions/BukkyOyetimehin/domManipulation/contact.js

```diff
@@ -0,0 +1,47 @@
+let submitForm = document.getElementById('validateForm');
+let fullName = document.getElementById('fullName');
+let email = document.getElementById('email');
+let subject = document.getElementById('subject');
+let message = document.getElementById('message');
+let fullNameRegex = /^[A-Za-z ]+$/;
+let emailRegex = /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/;
+let subjectRegex = /^[A-Za-z ]+$/;
+//Checking the form fields
+function required(field, regex, first_error, sec_error, event) {
+    if (field.value === "") {
+        field.nextElementSibling.innerHTML = first_error;
+        event.preventDefault();
+        return false;
+    } else if (!field.value.match(regex)) {
+        field.nextElementSibling.innerHTML = sec_error;
+        event.preventDefault();
+        return false;
+    } else {
+        field.nextElementSibling.innerHTML = "";
+        return true;
+    }
+}
+
+function checkmessage(message, event) {
```

Comment:
There is no need to pass `message` as a parameter here. Use it inside this function like that